src/lib.rs

```diff
@@ -55,12 +55,17 @@
 //! operation. When possible, it is a good idea to provide a size estimate by using the
 //! [`with_capacity`](HashMap::with_capacity) constructor. Note that using many keys with
 //! exactly the same [`Hash`](std::hash::Hash) value is a sure way to slow down performance of any
-//! hash table. /* TODO: dynamic load factor */
+//! hash table.
 //!
-//! /* TODO: set projection */
-//! /* TODO: frequency map through computeIfAbsent */
-//! /* TODO: bulk operations like forEach, search, and reduce */
+/*
```

Comment:
I'm not sure Rust will let you get away with breaking up a crate-level doc comment this way? You may have to do it the same way we did in the original version, with `/* */` around each individual thing, or with `//! /*` to start and `//! */` to end.

Reply:
The crate-level documentation is fine with this approach.
The reason I changed the comments is that the `TODO` blocks were displayed in the generated docs. I hope I'm not getting it wrong that you want the TODOs to be hidden from the user facing documentation, right ?
I joined a screenshot of the doc comment and the render to illustrate
![2020-02-04-002240_1920x1080_scrot](https://user-images.githubusercontent.com/11710698/73699725-d77cf580-46e4-11ea-907e-d85d094422a2.png)

Reply:
Oh, interesting, I was completely wrong then! Ignore me :sweat_smile: The behavior we want is that the TODOs are hidden from the docs, which it looks like your approach does!